static/src/js/components/SpatialDisplay/SpatialDisplay.js

```diff
@@ -331,6 +401,25 @@ class SpatialDisplay extends Component {
       : ['', '']
   }
 
+  /**
+   * Turns '1,2,3' into '2,1,3' for leaflet
+   * @param {String} circleCoordinates A center point and radius
+   */
+  transformCircleCoordinates(circleCoordinates) {
+    const points = circleCoordinates.split(',')
+
+    const lat = points[0] || ''
+    const lng = points[1] || ''
+    const radius = points[2] || ''
+
```

Comment:
Prefer destructuring

```suggestion
const [
  lat = '',
  lng = ''
  radius = ''
] = points
```